clicache/src/QueryService.hpp

```diff
@@ -22,6 +22,15 @@
 #include "begin_native.hpp"
 #include <geode/QueryService.hpp>
 #include "end_native.hpp"
+#include "Query.hpp"
+#include "Log.hpp"
+#include "CqAttributes.hpp"
+#include "CqQuery.hpp"
+#include "CqServiceStatistics.hpp"
+#include "impl/ManagedString.hpp"
+#include "ExceptionTypes.hpp"
+#include "impl/SafeConvert.hpp"
```

Comment:
Why are all these headers now needed?

Reply:
I *think they are, but I pulled their equivalents from the .cpp file, which they are not needed in.